linux/linux-6.6.defconfig

```diff
@@ -874,11 +888,22 @@ CONFIG_DRM_IMX_SEC_DSIM=y
 CONFIG_DRM_IMX_DCNANO=y
 CONFIG_DRM_IMX_DCSS=y
 CONFIG_DRM_IMX_CDNS_MHDP=y
+CONFIG_DRM_ETNAVIV=m
+CONFIG_DRM_HISI_HIBMC=m
+CONFIG_DRM_HISI_KIRIN=m
 CONFIG_DRM_MXSFB=y
 CONFIG_DRM_IMX_LCDIF=y
+CONFIG_DRM_PL111=m
+CONFIG_DRM_LIMA=m
+CONFIG_DRM_PANFROST=m
 CONFIG_FB=y
+CONFIG_FB_ARMCLCD=y
+CONFIG_FB_EFI=y
+CONFIG_FB_MXC_EINK_V2_PANEL=y
 CONFIG_BACKLIGHT_PWM=y
 CONFIG_BACKLIGHT_LP855X=m
+CONFIG_BACKLIGHT_GPIO=y
+CONFIG_BACKLIGHT_LED=y
```

Comment:
Any idea why the backlight stuff is in there?

Reply:
The M.2 expansion socket has pins for the LVDS interface. So LED display reasons.